lib/chai/assertion.js

```diff
@@ -138,11 +138,20 @@ module.exports = function (_chai, util) {
     if (!ok) {
       msg = util.getMessage(this, arguments);
       var actual = util.getActual(this, arguments);
-      throw new AssertionError(msg, {
-          actual: actual
+      var assertionErrorObjectProperties = {
+        actual: actual
```

Comment:
I know we should get auto-formatting in but this could perhaps be indented as before.